### DISCUSSION TITLE
NHL Centre Ice 1

### DISCUSSION THREAD

**Discussion by jphilippe2017** [iptv-org/community#742] [Q&A]
**Channel Name:** NHL Center Ice
**Channel ID:** NHLCenterIce1.us
**Country:** USA
**Language:** English
**Website:** https://www.nhl.com/info/center-ice
**Stream URL (optional):**
**Notes (optional):** HD please!

**Comment by AntiPontifex** [comment]
Dead issue: please close.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19732
